project_script/constantvalues.py

```diff
@@ -0,0 +1,9 @@
+electron_charge = 1.6*10**(-19)
+
+device_area = 1.4*10**(-5)
+
+device_thickness = 600*10**(-9)
+
+current_correction_factor = 1*10**(-9)
+
+charge_density_correction_factor = 1*10**(-6)
```

Comment:
In other languages, normally the constants is written in CAPS, to easily show that is a constant. Check if this practice is comon in python too.

Reply:
I checked in PEP8, and it's really in CAPS. Thanks!